packages/ui/src/molecules/Breadcrumb/Breadcrumb.test.tsx

```diff
@@ -95,6 +95,20 @@ describe('Breadcrumb', () => {
     expect(dividers).toHaveLength(2)
   })
 
+  it('Should not render null values', () => {
```

Comment:
WDYT about adding tests to check the last divider? Since you changed the value?

Reply:
I didn't change the behaviour, just the implementation, so tests that exists already test this